releases/stellar/2025-02-GMP-v1.1.0.md

```diff
@@ -0,0 +1,212 @@
+# Stellar GMP v1.1.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @nbayindirli (<noah@interoplabs.io>)       |
+| **Deployment** | @RiceAndMeet (<steven@interoplabs.io>) |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [v1.1.0 Crates](https://crates.io/crates/stellar-axelar-gateway/1.1.0)
+- [v1.1.0 Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases/tag/stellar-axelar-gateway-v1.1.0)
+
+## Background
+
+- This is the v1.1.0 Stellar GMP release.
+
+Contract changes in the release:
+
+1. Add contractstorage attribute macro, enabling named keys, typed values, storage method specification in the storage layout, storage schema validation, and built-in TTL extension. Use this macro in lieu of Soroban's contracttype macro on storage enums and calling storage on the env.
+2. Add only_owner and only_operator macros, reducing boilerplate for auth-checks and increasing code readability. Use this macro in lieu of directly checking the owner or operator.
+3. Migrate the axelar-operators contract's storage schema key from 'Operators' to 'Operator'.
+4. Add 'stellar/operators.js is-operator', 'stellar/operators.js add-operator', 'stellar/operators.js remove-operator', 'stellar/operators.js collect-fees', 'stellar/operators.js refund', and 'stellar/operators.js execute' commands to interact with the axelar-operators contract.
+
+Deployment changes in the release:
+
+1. Add 'stellar/gateway.js execute' command to execute GMP messages from the CLI.
+2. Make gas-token-address and gas-fee-amount optional for 'stellar/gmp.js send' command.
+3. Add 'stellar/contract.js paused', 'stellar/contract.js pause', 'stellar/contract.js unpause' commands to check the pausable state of a Stellar contract.
+4. Make minimum-rotation-delay optional for 'stellar/deploy-contract.js' commands.
+5. Add 'evm/gateway.js --action isContractCallApproved' command to verify if a gateway contract call is approved from the CLI.
+6. Add 'evm/gateway.js --action isMessageApproved' command to verify if a gateway message is approved from the CLI.
+
+## Deployment
+
+Create an `.env` config. `CHAIN` should be set to `stellar` for mainnet, and `stellar-2025-q1` for all other networks.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=xyz
+```
+
+An initial Stellar chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+### Devnet Amplifier / Stagenet / Testnet
+
+```bash
+\"$CHAIN\": {
```

Comment:
This is not a new deployment of Stellar, so this can be assumed to exist in the configs